src/lib/i18n/dictionaries/dictionary.json

```diff
@@ -3740,7 +3738,21 @@
   "crafting.readySoon": "{{name}} will be ready soon...",
   "crafting.expansionSoon": "Your land will be ready soon...",
   "crafting.viewRecipes": "View recipes",
-  "gems.buyReplenish": "Would you like to replenish the shops now for 20 Gems?",
+  "gems.buyReplenish": "Would you like to restock all the shops for 20 Gems?",
+  "gems.buyReplenish.enhanced": "Would you like to restock the {{shopName}} for {{gemPrice}} Gems?",
+  "restock.shop": "{{shopName}} Restock",
+  "restock.full": "Full Restock",
+  "restock.outOfStock": "Looks like you ran out of stock from {{npc}}!",
```

Comment:
I think we can shorten this + make it more direct: `Betty ran out of stock. Choose your desired restock option below:`